etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -168,11 +168,11 @@ prop.adminui.user.external_role_display=false
 # Name of the list to provide a user listing as used in the group editor of the admin ui.
 #
 # Values:
-# - USERS.INVERSE will list users by name (e.g. "John Sample")
-# - USERS.INVERSE.WITH.EMAIL will list users by name and email (e.g. "John Sample <j.sample@opencast.org>")
-# - USERS.INVERSE.WITH.USERNAME will list users by name and username (e.g. "John Sample (jsample)")
-# Default: USERS.INVERSE.WITH.USERNAME
-#prop.adminui.user.listname=USERS.INVERSE.WITH.USERNAME
+# - USERS.NAME will list users by name (e.g. "John Sample")
+# - USERS.NAME.AND.EMAIL will list users by name and email (e.g. "John Sample <j.sample@opencast.org>")
+# - USERS.NAME.AND.USERNAME will list users by name and username (e.g. "John Sample (jsample)")
+# Default: USERS.NAME.AND.USERNAME
+prop.adminui.user.listname=USERS.NAME.AND.EMAIL
```

Comment:
Please don't change the default configuration

Reply:
Whoops, this wasn't intended. Fixed!